cookbooks/ambari/spec/unit/recipes/ambari_agent_setup_spec.rb

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Why is this commented?

Reply:
ambari-agent's are not required, it might be clean to remove the file.